### ISSUE TITLE
Does gptSessionBenchmark benchmark tool support prompt text as input?

### ISSUE THREAD

**Issue by Fred-cell** [NVIDIA/TensorRT-LLM#906]
Usage:
  TensorRT-LLM C++ Runtime Benchmark [OPTION...]

  -h, --help                    Print usage
  -m, --model arg               Model name specified for engines. (default:
                                gpt_350m)
      --engine_dir arg          Directory that store the engines.
      --batch_size arg          Specify batch size(s) you want to
                                benchmark. Multiple batch sizes can be
                                separated by ";", example: "1;8;64".
                                (default: 8)
      --beam_width arg          Specify beam width you want to benchmark.
                                (default: 1)
      --input_output_len arg    Specify input-output length(s) you want to
                                benchmark. Multiple input lengths can be
                                separated by ";", example: "60,20;128,20".
                                (default: 128,20)
      --log_level arg           Choose log level between
                                verbose/info/warning/error/internal_error.
                                (default: error)
      --warm_up arg             Specify warm up iterations before benchmark
                                starts. (default: 2)
      --num_runs arg            Minimal number of iterations to run during
                                benchmarking. (default: 10)
      --duration arg            Minimal duration of iterations to measure
                                in seconds. (default: 60)
      --ctx_micro_batch_size arg
                                Batch size for context phase.
      --gen_micro_batch_size arg
                                Batch size for generation phase.
      --max_attention_window arg
                                Max kv cache length per sequence.
      --max_tokens_in_paged_kvcache arg
                                Max tokens in paged K-V Cache.
      --kv_cache_free_gpu_mem_fraction arg
                                K-V Cache Free Gpu Mem Fraction.
      --enable_cuda_graph       Execute GPT session with CUDA graph.
      --print_all_logits        Print all context and generation logits.

```[tasklist]
### Tasks
```

**Comment by byshiue** [comment]
It is not supported. `gptSessionBenchmark` does not contain the tokenizer. It will use random input ids as inputs.
